docs/core-concepts/websocket/index.md

```diff
@@ -41,14 +37,14 @@ On the other hand, `wss://` is WebSocket over TLS, (same as HTTPS is HTTP over T
 
 Once the socket is created, we should listen to events on it. There are totally 4 events:
 
-- open – connection established,
-- message – data received,
-- error – websocket error,
-- close – connection closed.
+* open – connection established,
+* message – data received,
+* error – websocket error,
+* close – connection closed.
 
-…And if we’d like to send something, then socket.send(data) will do that.
+And if we’d like to send a message then socket.send(data) will do that.
 
-Here’s an example:
+Here’s an example in `Javascript`:
 
 ```js
 const app_id = 1089; // Replace with your app_id or leave as 1089 for testing.
```

Comment:
```suggestion
const app_id = 1089; // Replace with your app_id or leave as 1089 for testing.
const socket = new WebSocket(`wss://ws.binaryws.com/websockets/v3?app_id=${app_id}`);

socket.onopen = function (e) {
   console.log('[open] Connection established');
   console.log('Sending to server');
  const sendMessage = JSON.stringify({ ping: 1 });
  socket.send(sendMessage);
};

socket.onmessage = function (event) {
   console.log(`[message] Data received from server: ${event.data}`);
};

socket.onclose = function (event) {
  if (event.wasClean) {
     console.log(`[close] Connection closed cleanly, code=${event.code} reason=${event.reason}`);
  } else {
    // e.g. server process killed or network down
    // event.code is usually 1006 in this case
     console.log('[close] Connection died');
  }
};

socket.onerror = function (error) {
   console.log(`[error]`);
};
```
The block of code had some bugs that are fixed and I think `console.log` should be better than `alert` here